docs/source/docs/get-started/quick-start-guide/examples.rst

```diff
@@ -0,0 +1,98 @@
+Examples
+########
+
+- Convert PASCAL VOC dataset to COCO format, keep only images with ``cat`` class
+  presented:
+
+.. code-block::
+
+    # Download VOC dataset:
+    # http://host.robots.ox.ac.uk/pascal/VOC/voc2012/VOCtrainval_11-May-2012.tar
+    datum convert --input-format voc --input-path <path/to/voc> \
+                  --output-format coco \
+                  --filter '/item[annotation/label="cat"]' \
+                  -- --reindex 1 # avoid annotation id conflicts
+
+- Convert only non-``occluded`` annotations from a
+  `CVAT <https://github.com/openvinotoolkit/cvat>`_ project to TFrecord:
```

Comment:
This might be redundant for now.

Reply:
> This might be redundant for now.

I corrected all old cvat repo URL in docs.